solr/config/configsets/all_collection_oag56_v1/managed-schema

```diff
@@ -286,6 +286,7 @@
       <filter class="solr.LowerCaseFilterFactory"/>
     </analyzer>
   </fieldType>
+  <fieldType name="text_general_exportation" class="solr.TextField" omitNorms="true" indexed="false" stored="true" uninvertible="false" docValues="false"/>
```

Comment:
For the future - use more generic names. It's not about exportation, in the big picture is about not indexed text field. It can stay as it is

Reply:
We have discussed this topic before. I apologize for the confusion. I have updated `text_general_exportation` to `text_general_not_indexed` to clarify the intended functionality.